lib/Translation/ExportVerilog/ExportVerilog.cpp

```diff
@@ -20,6 +20,7 @@
 #include "circt/Dialect/SV/SVVisitors.h"
 #include "circt/Support/LLVM.h"
 #include "circt/Support/LoweringOptions.h"
+#include "mlir/IR/AsmState.h"
```

Comment:
I don't think AsmState.h is needed?

Reply:
Ah, must have automatically been imported by Neovim for something I removed and then I forgot about it, sorry!